htdocs/core/lib/functions2.lib.php

```diff
@@ -814,7 +814,7 @@ function get_next_value($db,$mask,$table,$field,$where='',$objsoc='',$date='',$m
     //print "masktri=".$masktri." maskcounter=".$maskcounter." maskraz=".$maskraz." maskoffset=".$maskoffset."<br>\n";
 
     // Define $sqlstring
-    $posnumstart=strrpos($maskwithnocode,$maskcounter);	// Pos of counter in final string (from 0 to ...)
+    $posnumstart=mb_strrpos($maskwithnocode,$maskcounter, 'UTF-8');	// Pos of counter in final string (from 0 to ...)
```

Comment:
Yes, you can fix the issue locally by overloading regular functions with mbstring enabled ones.
But that doesn't change the fact that if this PR is accepted as-is, it's breaking all non mbstring enabled installs by calling an undefined function.
I suggest that kind of code instead:
```php
 // Use the multibyte enabled function if available or fallback to the regular one
if (function_exists('mb_strrpos')) {
    $posnumstart=mb_strrpos($maskwithnocode,$maskcounter, 'UTF-8');
} else {
    $posnumstart=strrpos($maskwithnocode,$maskcounter);
}
```